install/kubernetes/helm/istio/charts/galley/values.yaml

```diff
@@ -29,3 +29,5 @@ tolerations: []
 # "security" and value "S1".
 podAntiAffinityLabelSelector: []
 podAntiAffinityTermLabelSelector: []
+#HTTPS port of the validation service
```

Comment:
Please use it as default value in code, as it is used in a lot of other places.

We do not need to add a user option for every internal port or thing that shows up in a template.

Reply:
thanks for your comment. have a PR https://github.com/istio/istio/pull/16015 to address.